### DISCUSSION TITLE
Failed to resolve versions

### DISCUSSION THREAD

**Discussion by th3hybrid** [tinchoabbate/damn-vulnerable-defi#83] [Help / Q&A / Troubleshooting]
when I started, I cloned the whole repo and completed the first challenge. I want to build and test now, but I have version incompatibility with some contracts in the lib. I just wanted to know how everyone handles theirs.

**Comment by AlexQllx** [comment]
I had an old version of foundry so I did a `foundryup` then re-run the `forge build` which installed my missing solc version for 0.8.25
